autopush/endpoint.py

```diff
@@ -253,22 +253,25 @@ def _router_fail_err(self, fail):
         fail.trap(RouterException)
         exc = fail.value
         if isinstance(exc, RouterException):
```

Comment:
can you kill this isinstance check while you're here? it's useless since we already trapped it